encfs/FileUtils.cpp

```diff
@@ -1739,6 +1739,9 @@ void unmountFS(const char *mountPoint) {
   // fuse_unmount does not work on Mac OS, see #428
   unmount(mountPoint, MNT_FORCE);
 #endif
+#ifdef __CYGWIN__
+  system(string("/usr/bin/pkill -f '(^|/)encfs .*/.* ").append(mountPoint).append("?( |$)'").c_str());
```

Comment:
Shell injection vulnerabilty!

Reply:
Thank you for your review 👍 
I think a regex match of `mountPoint` would do the trick ?
Perhaps you thought about something else ?

Reply:
The best way is to avoid the shell by using something like execl

Reply:
Done 👍